### ISSUE TITLE
Create a text widget

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#61]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To enter text into pages.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via b71d92de87d63c4b766ca17bb708c1abdcb6da57
